### PR TITLE
Deleted Itinerary Model and Fixed Typo for Description

### LINKED ISSUE #115: Refactor Stage And Trip Attribute Names

### PR DESCRIPTION
**Pull request by AndyLiang1**

### CODE REVIEW

**Review by L0Lmaker — APPROVED** [review]

File: server/models/StageModel.js [hunk 1]
```diff
@@ -24,8 +20,9 @@ const stageSchema = new mongoose.Schema({
   stageLongitude: Number,
   stageLatitude: Number,
   locationName: String,
```
Comment (vee-16) [inline server/models/StageModel.js]: Just adding a suggestion, what do you think if we further clarified the following attributes for Stage/Trip Models:
 
StageModel.js
- locationName -- change to --> stageLocation
- stageLongitude
- stageLatitude

TripModel.js
- destination -- change to --> tripLocation
- tripLongitude
- tripLatitude

**Comment by vee-16** [comment]
closes #115

**Comment by L0Lmaker** [comment]
too many changes in this PR, I will make separate ones for each of these changes

**Comment by L0Lmaker** [comment]
Closing as PR split up

**Pull request closed without merging**